examples/subscribers/update.js

```diff
@@ -0,0 +1,40 @@
+// node examples/subscribers/update.js
+
+"use strict";
+
+import dotenv from 'dotenv';
+import MailerLite from '../../dist/index.mjs';
+
+dotenv.config();
+
+const mailerlite = new MailerLite({
+  api_key: process.env.API_KEY,
+});
+
+const subscriberId = '<subscriber_id>';
```

Comment:
Following the patter from other examples it should be
1. `// node examples/subscribers/update.js ID_string`

2.
```suggestion
let subscriber_id = null
if (process.argv.slice(2).length) {
    subscriber_id = process.argv[2];
}
```

3. `mailerlite.subscribers.update(subscriber_id,subscriberData)`